cirq/ops/common_gates.py

```diff
@@ -116,6 +116,30 @@ def _trace_distance_bound_(self) -> Optional[float]:
             return None
         return abs(np.sin(self._exponent * 0.5 * np.pi))
 
+    def controlled(self,
+                   num_controls: int = None,
+                   control_values: Optional[Sequence[
+                       Union[int, Collection[int]]]] = None,
+                   control_qid_shape: Optional[Tuple[int, ...]] = None
+                  ) -> raw_types.Gate:
+        """
+        Specialize controlled method for XPowGate to construct a corresponding
+        controlled CNotPowGate, if applicable.
+        Specialization only happens when the last specified control (which acts
+        first semantically) is a default-type control qubit.
```

Comment:
Actually, now that I am writing an example pydoc, what does "default-type" control qubit mean?  Is this just a qubit and not a qudit?

Reply:
It means a control that is satisfied by the qubit being ON, as opposed to OFF. Also the qubit vs qutrit thing.